js/views/adScreen.js

```diff
@@ -135,8 +135,10 @@ var AdScreen = React.createClass({
     };
 
     var playbackControlItems = [];
-    for(var item in playbackControlItemTemplates) {
-      playbackControlItems.push(playbackControlItemTemplates[item]);
+    for (var item in playbackControlItemTemplates) {
```

Comment:
Wouldn't for (var item = 0; item < playbackControlItemTemplates.length; item++) be more effective?